README.md

```diff
@@ -76,6 +76,12 @@ Every page can override config options. You can pass theme as a third parameter
 {{ photoswipe('id', 'a', page.header.photoswipe.options) }}
 ```
 
+> **If you call the Twig photoswipe function from a modular template**, you must add this to the modular page's frontmatter: `never_cache_twig: true`. You don't need to do this if your site has caching disabled.
```

Comment:
Would you mind changing the bold part a bit? Because modular page is the main page containing all the separate modules.
```suggestion
> **If you use Photoswipe in a module of the modular page template**, you must add this to the module page's (not the main modular page) frontmatter: `never_cache_twig: true`. You don't need to do this if your site has caching disabled.
```

Reply:
It's completely up to you.

I don't agree with your change because the wording seems Quark-specific, or at least assumes (or implies that) the template is called `modular`. Or I'm confused.

I think terminology around modules is confusing, but my understanding is:

* "module" is the new preferred term (or object name in code) for the modular components of pages
* "modular pages" use modules that typically sit beneath them.

I think "a modular template" that I wrote in the bolded section should be "a module template".

I think you corrected "the modular page's frontmatter" but maybe it's even better as "the module's frontmatter"??

Sometimes I feel like a diagram is needed :/

Reply:
I don't know how to edit your change suggestion directly, so I'll paste my new suggestion here:

```markdown
> If you use Photoswipe in a module template**, you must add this to the module's (not the main modular page) frontmatter: `never_cache_twig: true`. You don't need to do this if your site has caching disabled.
```

Another suggestion because I can't stop tweaking :) ..
```markdown
> If you use Photoswipe in a module template**, you must add this to frontmatter in the `.md` file of the module (not the main modular page): `never_cache_twig: true`. You don't need to do this if your site has caching disabled.
```

Reply:
My reasoning is based on [the docs of modular page](https://learn.getgrav.org/17/content/modular) and I think you got it right - modular contains modules. Issue with plugin is when it's rendered in a module. No issues if it's in a modular (main container).

Your first suggestion of the two (with the frontmatter) is more clear to me I think